base/deprecated.jl

```diff
@@ -1581,6 +1581,10 @@ end
 
 @deprecate String(io::GenericIOBuffer) String(take!(copy(io)))
 
+
+@deprecate readstring(s::IO) read(s, String)
+@deprecate readstring(filename::String) read(filename, String)
+@deprecate readstring(cmd::AbstractCmd) read(cmd, String)
```

Comment:
Perhaps nix the extra empty line above these definitions and restore the empty line before `# END ...`?